packages/storage/src/indexed-db/index.ts

```diff
@@ -388,12 +392,16 @@ export class IndexedDb implements IndexedDbInterface {
     return SwapRecord.fromJson(json);
   }
 
-  async getGasPrices(): Promise<GasPrices | undefined> {
-    return this.db.get('GAS_PRICES', 'gas_prices');
+  async getGasPrices(): Promise<PlainMessage<GasPrices> | undefined> {
+    return await this.db.get('GAS_PRICES', 'gas_prices');
```

Comment:
Can we serialize/deserialize through json here?

Reply:
Also, think `GasPrices` type is better to passing around the application

Reply:
this seems to just work finely